test/functional/features/overview-page.feature

```diff
@@ -3,7 +3,22 @@ Feature: Overview page
   As a citizen
   I want to know what stage of the appeal I am at
 
-  Scenario: Entering task-list from overview page
-    Given I am authenticated as a valid appellant
+  Scenario: Appeal Started
+#    New Appeal Started
+    Given I have logged in as an appellant in state "New appealStarted"
```

Comment:
Bit of me being anal but if this is what you want to test it should be When not given